src/sentry/auth/helper.py

```diff
@@ -62,6 +62,8 @@
 
 ERR_INVALID_IDENTITY = _("The provider did not return a valid user identity.")
 
+_REDIS_KEY = "verificationKeyStorage"
```

Comment:
you can import this key from your other file. it is probably best to actually abstract all of  this away in that file so helper doesn't need to know about those details.